database/src/migrations/release.0.5/db_setup_up.sql

```diff
@@ -40,51 +40,51 @@ ALTER DEFAULT PRIVILEGES IN SCHEMA restoration GRANT ALL ON TABLES TO restoratio
 
 alter role restoration_api set search_path to restoration_dapi_v1, restoration, public, topology;
 
-\i restoration.sql
-\i populate_user_identity_source.sql
-\i api_set_context.sql
-\i tr_audit_trigger.sql
-\i tr_generated_audit_triggers.sql
-\i api_get_context_user_id.sql
-\i api_get_context_system_user_role_id.sql
-\i api_user_is_administrator.sql
-\i tr_journal_trigger.sql
-\i tr_generated_journal_triggers.sql
-\i tr_project_funding_source.sql
-\i tr_project.sql
-\i tr_permit.sql
-\i api_get_system_constant.sql
-\i api_get_system_metadata_constant.sql
+-- \i restoration.sql
+-- \i populate_user_identity_source.sql
+-- \i api_set_context.sql

```

Comment:
Does this all still need to be commented out?

As far as our migrations go, we don't actually use this particular sql file (its all copied over to the migration file itself), but not sure if this has any impact outside of our migrations (probably not?)

Reply:
I do not know. I played around with this file a bit until I realized it wasn't actually used.

Reply:
I still need to make this work though :)

Reply:
gotcha, no worries